docs/_includes/status.html

```diff
@@ -0,0 +1,20 @@
+{%- assign url_parts = page.url | split: '/' %}
+{%- assign spec_name = url_parts[1] %}
+{%- assign spec_version = url_parts[2] %}
+{%- assign spec_version_num = spec_version | remove: 'v' %}
+{%- assign current_version = site.data.versions[spec_name].current %}
+{%- assign current_version_num = current_version | remove: 'v' %}
+
+{%- if site.data.versions[spec_name].versions[spec_version].status %}
+
+{%- if page.layout == "specifications" %}
+<div class="rounded-lg p-4 border border-gray-400 mt-8 md:mt-0 mb-4">
+{%- else %} <!-- page.layout == "standard" -->
+<div class="rounded-lg p-4 border border-green-900 mt-8 md:mt-0 mb-4">
```

Comment:
No need for the `mt` thing - it just adds extra space on mobile, which looks bad.

```suggestion
<div class="rounded-lg p-4 border border-green-900 mb-4">
```

Reply:
Ah. Good to know what these are about. Thanks.